tests/integration/docusaurus/tutorials/quickstart/quickstart.py

```diff
@@ -10,33 +14,34 @@
 
 # Connect to data
 # <snippet name="tutorials/quickstart/quickstart.py connect_to_data">
-validator = context.sources.pandas_default.read_csv(
+batch = context.sources.pandas_default.read_csv(
     "https://raw.githubusercontent.com/great-expectations/gx_tutorials/main/data/yellow_tripdata_sample_2019-01.csv"
 )
 # </snippet>
 
 # Create Expectations
 # <snippet name="tutorials/quickstart/quickstart.py create_expectation">
-validator.expect_column_values_to_not_be_null("pickup_datetime")
-validator.expect_column_values_to_be_between(
-    "passenger_count", min_value=1, max_value=6
+suite = context.add_expectation_suite("my_suite")
+
+# TODO: update where these expectations are imported
```

Comment:
I'm tracking removing these TODOs in https://greatexpectations.atlassian.net/browse/V1-111